src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -1016,6 +1021,10 @@ static LogicalResult getVariadicSimdCodeFullyFlattened(
   Type outputElementType = outputMemRefType.getElementType();
   unsigned numArgs = op->getNumOperands();
 
+  if (DEBUG)
+    fprintf(stderr, "hi alex, simdizing variadic op %s\n",
```

Comment:
Please make sure to remove this "hi, alex" :)

Reply:
Ohhh, but I want to be famous :-)